plugins/com.google.cloud.tools.eclipse.ui.util/src/com/google/cloud/tools/eclipse/ui/util/messages.properties

```diff
@@ -0,0 +1,10 @@
+bucket.name.invalid=Invalid bucket name
```

Comment:
Maybe not in this PR, but in general it's useful to be very specific about failures of this kind. E.g. tell the user that the bucket name is too long, too short, or has invalid characters in three separate messages. And tell them which characters are invalid in the message.